Userland/Libraries/LibWeb/DOM/HTMLFormControlsCollection.cpp

```diff
@@ -77,4 +77,11 @@ Variant<Empty, Element*, JS::Handle<RadioNodeList>> HTMLFormControlsCollection::
     }));
 }
 
+WebIDL::ExceptionOr<JS::Value> HTMLFormControlsCollection::named_item_value(FlyString const& name) const
+{
+    return const_cast<HTMLFormControlsCollection&>(*this).named_item_or_radio_node_list(name).visit(
```

Comment:
This const_cast is a bit yuck, it's because `named_item_or_radio_node_list` passes the `root()` to RadioNodeList, which needs it to be non-const for the GC-visit. Any better ideas for this?

Reply:
How about we add const pointer overloads for the visit functions? :yakthonk:

Reply:
Just started looking into that and realised, I think I can just change the constness of `m_root` in `LiveNodeList`.

I didn't think there were const overloads, but I guess it must be because of the:
```
        void visit(NonnullGCPtr<T> cell)
        {
            visit_impl(const_cast<RemoveConst<T>&>(*cell.ptr()));
        }
```

In `Visitor`

Reply:
Done - but I'm also wondering from this why all of the overloads accept a const-pointer if we accept a `NonnullGCPtr<const T>` :yakthonk: